src/nice-path-decomposition/nice-path-decomposition.h

```diff
@@ -0,0 +1,33 @@
+#include "../centroid-tree.h"
+
+
+class NicePathDecomposition : public PathDecomposition {
```

Comment:
Add a docstring: a definition of a nice path decomposition and the purpose of this class.